### PR TITLE
Client Functionality and CSS

### PR DESCRIPTION
**Pull request by beewinn**

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
 const express = require("express");
 const path = require("path");
 var router = require("./router.js");
+var model = require("./model.js");
```
Comment (dpars88) [inline server/server.js]: Do you need this in this file? It doesn't seem to be used at all. Not sure if this is for future work or not.

File: server/router.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 var express = require("express");
 var model = require("./model.js");
 var router = express.Router();
+const path = require("path");
```
Comment (dpars88) [inline server/router.js]: Not sure you need path here either. But also not sure if you're planning ahead.

File: client/src/app.jsx [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+		this.handleReserveClick = () => {
+			var parts = document.URL.split("/");
+			var lastSegment = parts.pop() || parts.pop(); // handle potential trailing slash
+			var data = {
+				id: lastSegment,
+				checkIn: this.state.selectedFirstDate,
+				checkOut: this.state.selectedSecDate
+			};
+
+			//post request logic pending
```
Comment (dpars88) [inline client/src/app.jsx]: I appreciate the comments in this file! Helps me stay focused on the function I'm reading

File: server/server.js (hunk 1 above)
Comment (beewinn) [reply server/server.js]: I resolve this in my next PR, thanks for catching!

File: server/router.js (hunk 2 above)
Comment (beewinn) [reply server/router.js]: I resolve this in my next PR, thanks for catching!

**Pull request merged**